experiments/compositions/composition/tests/utils/utils.go

```diff
@@ -84,10 +85,30 @@ func GetUnstructuredObj(g, v, k, ns, n string) *unstructured.Unstructured {
 	return &cr
 }
 
+func GetCompositionObj(ns, n string) *unstructured.Unstructured {
+	return GetUnstructuredObj("composition.google.com", "v1", "Composition", ns, n)
+}
+
 func GetPlanObj(ns, n string) *unstructured.Unstructured {
 	return GetUnstructuredObj("composition.google.com", "v1", "Plan", ns, n)
```

Comment:
We probably should make these v1alpha1 asap

Reply:
+1 in a separate PR

Reply:
https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1530